### ISSUE TITLE
[1.18.1-Forge] Crash when entering world. (Cross issue with Balm Mod?)

### ISSUE THREAD

**Issue by Gbergz** [refinedmods/refinedstorage#3174]
### Describe the bug

When entering a world with Refined Storage + Maybe balm? the game crashes.

### How can we reproduce this bug or crash?

1. Install Refined Storage.
2. Install Balm (maybe).
3. Enter world.
4. Crash.

### What Minecraft version is this happening on?

Minecraft 1.18.1

### What Forge version is this happening on?

39.0.5

### What Refined Storage version is this happening on?

1.10.0-beta.1

### Relevant log output

```shell
Crash-report: https://gist.github.com/Gbergz/ff017bbe072cb11dfeda7030c9a61459
latest.log: https://gist.github.com/Gbergz/79f2b7a91a9339e330b6844dab99a54a
```

**Comment by raoulvdberge** [comment]
Try without Balm before creating an issue?

**Comment by raoulvdberge** [comment]
On first sight, just by looking at the stack trace, the issue seems to be caused by Crafting Tweaks. But I could be wrong.

@blay09 What do you think?

**Comment by Gbergz** [comment]
With just Refined Storage it works.
With Refined Storage + Balm it works.

With Refined Storage + Balm + CraftingTweak it also works. 

So right now I'm thinking its clashing with some other mod that is perhaps using Balm?

**Comment by blay09** [comment]
Fixed in Crafting Tweaks 14.0.1
